### DISCUSSION TITLE
SPA section

### DISCUSSION THREAD

**Discussion by ap1969** [quasarframework/quasar#9760] [Ideas / Proposals] [answered]
Hi,
Stupid question - but there's no "CLI - SPA" mode. Any reason for that?
Andy

**Comment by rstoenescu** [comment, marked as answer]
Hi. It's the default mode. https://quasar.dev/quasar-cli/developing-spa/introduction

**Reply by ap1969** [reply]
Hi, sorry, I was meaning in this forum. If I'm developing an SPA, where do I ask questions?

Andy

**Reply by MilosPaunovic** [reply]
@ap1969 This is the [Quasar Forum](https://forum.quasar-framework.org/) which might serve your purpose better.

**Reply by ontwikkelfabriek** [reply]
@MilosPaunovic  @ap1969 @rstoenescu 

No that forum is closed and dead. This is the replacement for that forum. I also think an SPA section is missing , 95% of all the Quasar apps are build in SPA mode and there's no section in this 'forum' and that is very weird.

**Reply by rstoenescu** [reply]
There is a limited amount of categories that GH Discussions supports. And we've hit that limit.
SPA is indeed the most basic one. There's nothing special to it. Just a page with components on it, so that's why it fell short when deciding categories (along with the limited number of categories). Most "discussions" about SPA are actually discussions about components/directives/plugins. So please use that category.

**Reply by ap1969** [reply]
Ah, I'm surprised there's a limit. OK, thanks for getting back to me.

**Reply by hawkeye64** [reply]
Apparently, you can only have a max of 10 categories per repo.

**Reply by ontwikkelfabriek** [reply]
@hawkeye64 @rstoenescu 

Still happy about the decision of dropping the forum in favor of Github discussions? I think there's a LOT less activity / knowledge exchange here compared to the old forum.

**Reply by rstoenescu** [reply]
Given a bit more time there won't be any difference in activity. With any change, there are of course upsides and downsides. The upsides outweight the downsides in my opinion. For example: the better integration with Github (can link github issues easily; can transform issues to discussion or vice-versa) and you now need only one account (Github) instead of a specific forum one. Also, this forum has more visibility given it's right there in the top menu of the GH repo. We also don't need to spend additional time (and money) maintaining the forum server with its software (better use that time to code for Quasar itself).

**Reply by rstoenescu** [reply]
Please do connect with me on Discord for any ideas on how we can improve this forum.
